roles/ceph-mon/tasks/create_mds_filesystems.yml

```diff
@@ -22,12 +22,11 @@
   changed_when: false
   when:
     - ceph_release_num[ceph_release] >= ceph_release_num.jewel
-    - mds_allow_multimds
+    - ceph_release_num[ceph_release] < ceph_release_num.mimic
```

Comment:
from what I understand you want to skip this task only for releases prior to jewel, right?
is that something that will disappear after mimic ?

Reply:
Yes, allow_multimds will be deprecated in mimic and will disappear afterward (as multiple MDS will always be allowed). It was not stable prior to jewel, so this commit keeps ceph-ansible's prior behavior of not enabling it for versions prior to jewel.